types/mindustry/index.d.ts

```diff
@@ -0,0 +1,294 @@
+    send(object: any): void;
+}
+ export class UDPConnection {
+    #writeBuffer: DataStream;
+    #serializer: PacketSerializer;
+    #udp: dgram.Socket;
+    #timer: NodeJS.Timer;
+    #connected: boolean;
+    constructor(w: number, s: PacketSerializer, p: (Packet: Packet) => void);
+    on(name: string, func: (...args: any[]) => void): void;
+    connect(port: number, ip: string): void;
+    close(): void;
+    readObejct(d: unknown): null | Error | object;
+    send(object: any): void;
+}
+// tslint:disable-next-line:no-unnecessary-class
+ export class FrameworkMessage {
+    static RegisterTCP: Class<unknown>;
+    static KeepAlive: Class<unknown>;
+    static RegisterUDP: Class<unknown>;
+}
+
+ export class PacketSerializer {
+    #temp: DataStream;
+    constructor();
+    read(buf: unknown): void | Packet;
+    write(buf: DataStream, object: any): void;
+    writeLength(buf: DataStream, len: number): void;
+    writeFramework(buf: DataStream, msg: FrameworkMessage): void;
+    readFramework(buf: DataStream): void;
+}
+ export class Client {
+    #UDPRegistered: boolean;
+    #TCPRegistered: boolean;
+    #tcp: TCPConnection;
+    #udp: UDPConnection;
+    #event: EventEmitter;
+    #parser: (Packet: Packet) => void;
+    constructor(w: number, s: PacketSerializer, p: (Packet: Packet) => void);
+    close(): void;
+    connect(port: number, ip: string): void;
+    on(name: string, func: (...args: any[]) => void): void;
+    once(name: string, func: (...args: any[]) => void): void;
+    sendTCP(obj: any): void;
+    sendUDP(obj: any): void;
+    parse<T extends Packet>(Packet: T): void;
+}
+
+ export class StreamBuilder {
+    id: number;
+    type: unknown;
+    total: unknown;
+    stream: Buffer;
+    constructor(packet: unknown);
+    add(data: any): void;
+    isDone(): boolean;
+    build(): Packet;
+}
+// tslint:disable-next-line:no-unnecessary-class
+ export class NetworkIO {}
+// tslint:disable-next-line:no-unnecessary-class
+ export class SaveIO {
+    static readStringMap(buf: DataStream): Map<string, string>;
+}
+ export class NetClient {
+    #client: Client;
+    #event: EventEmitter;
+    #streams: Map<number, StreamBuilder>;
+    game: unknown;
+    constructor(game?: unknown);
+    connect(port: number, ip: string): void;
+    reset(): void;
+    on(name: string, func: (...args: any[]) => void): void;
+    once(name: string, func: (...args: any[]) => void): void;
+    send(packet: any, reliabale: boolean): void;
+    join(name: string, uuid: string, usid: string): void;
+    connectConfirm(): void;
+    sendChatMessage(msg: string): void;
+    handleClientReceived(packet: Packet): void;
+    loadWorld(packet: unknown, game: unknown): void;
+}
+export function pingHost(port: string, ip: string, callback: (p: unknown, err?: Error) => void): void;
+ export class Mindustry {
+    netClient: NetClient;
+    constructor();
+}
+// Definitions by: EmmmM9O <https://github.com/EmmmM9O>
```

Comment:
???
can you remove thos comments, unless part of the header (so move them to the header)
Please also run formatter on your code:
https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes